app/views/courses/show.html.erb

```diff
@@ -5,127 +5,198 @@
     set_meta_tags image_src: @course.image
   %>
 
-  <section class="course-page container">
-    <div class="row justify-content-around">
+  <div class="row mx-5">
+    <nav class="col-8 px-0 d-sm-none" aria-label="breadcrumb">
+      <ol class="breadcrumb my-auto px-0">
+        <li class="breadcrumb-item"><a href="<%= courses_path -%>">&lt;&lt; Pathways</a></li>
+      </ol>
+    </nav>
+    <nav class="col-8 d-sm-inline d-none" aria-label="breadcrumb">
+      <ol class="breadcrumb my-auto px-0">
+        <li class="breadcrumb-item"><a href="/">Home</a></li>
+        <li class="breadcrumb-item"><a href="<%= courses_path -%>">Pathways</a></li>
+        <li class="breadcrumb-item active" aria-current="page"><%= tc(@course.name) %></li>
+      </ol>
+    </nav>
+    <div class="col-4 px-0 my-auto text-right">
+      <button class="btn btn-primary dropdown-toggle" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+        <%= LocalesService.language_name(params[:locale]) %>
+      </button>
+      <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
```

Comment:
Recommend adding the `dropdown-menu-right` class here, which forces the dropdown menu that opens to be right aligned (makes for a better layout/UI, especially on mobile).